docs/content/users/configuration/config_yaml.md

```diff
@@ -1,76 +1,561 @@
+
+## `router_http_port`
+
+Port for DDEV router’s HTTP traffic.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-file-directory-16: project | `80` | Usually changed only if there’s a conflicting process using that port.
+
+See the [Troubleshooting](../basics/troubleshooting.md#web-server-ports-already-occupied) page for more on addressing port conflicts.
+
+## `router_https_port`
+
+Port for DDEV router’s HTTPS traffic.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-file-directory-16: project | `443` | Usually changed only if there’s a conflicting process using that port.
+
+See the [Troubleshooting](../basics/troubleshooting.md#web-server-ports-already-occupied) page for more on addressing port conflicts.
+
+## `simple-formatting`
+
+Whether to disable most `ddev list` and `ddev describe` table formatting.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-globe-16: global | `false` | Can be `true` or `false`.
+
+When `true`, turns off most table formatting in `ddev list` and `ddev describe` and suppresses colorized text everywhere.
+
+## `table-style`
+
+Style for `ddev list` and `ddev describe`.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-globe-16: global | `default` | Can be `default`, `bold`, and `bright`.
+
+`bright` is a pleasant, colorful output some people may prefer.
+
+## `timezone`
+
+Timezone for container and PHP configuration.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-file-directory-16: project | `UTC` | Can be any [valid timezone](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), like `Europe/Dublin` or `MST7MDT`.
+
+## `type`
+
+The DDEV-specific project type.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-file-directory-16: project | `php` | Can be `backdrop`, `craftcms`, `drupal6`, `drupal7`, `drupal8`, `drupal9`, `drupal10`,  `laravel`, `magento`, `magento2`, `php`, `shopware6`, `typo3`, or `wordpress`.
+
+The `php` type doesn’t attempt [CMS configuration](../../users/quickstart.md) or settings file management and can work with any project.
+
+## `upload_dir`
+
+Path from the project’s docroot to the user-generated files directory targeted by `ddev import-files`.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-file-directory-16: project | |
+
+This is a CMS-specific setting that won’t have any effect for generic `php` [project types](#type).
+
+## `use_dns_when_possible`
+
+Whether to use DNS instead of editing `/etc/hosts`.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-file-directory-16: project | `true` | Can be `true` or `false`.
+
+When `false`, DDEV will always update the `/etc/hosts` file with the project hostname instead of using DNS for name resolution.
+
+See [Using DDEV Offline](../details/offline-usage.md).
+
+## `use_docker_compose_from_path`
+
+Whether to use the system-installed docker-compose. You can otherwise use [`required_docker_compose_version`](#required_docker_compose_version) to specify a version for download.
```

Comment:
This should only be used in specific troubleshooting scenarios.

Reply:
[Fixed](https://github.com/drud/ddev/pull/4332/commits/5258a6541398d0521dde8f64b5b151bacbf53191).